packages/js/components/changelog/add-35851-tree-control-highlight-item

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Add hightlighter to the tree control
```

Comment:
Typo here! `highlighter` instead of `hightlighter`

Reply:
Ohh! Nice catch. Thanks!